blog/_posts/2025-08-05-aks-mcp-server.md

```diff
@@ -0,0 +1,118 @@
+affinity for Claude code or Github-Copilot or some other agent for your workflows,
+you can plug-in AKS-MCP Server and unlock intelligent interactions, automation and troubleshooting for your AKS environments.
+
+![aks-mcp-github](/assets/images/aks-mcp/aks-mcp-github.png)
+
+## The Problem: Why Do We Need MCP Now?
+
+The biggest pain point facing modern AI assistants is not their reasoning or language abilities,
+but the fragmented, brittle context in which they operate. Organizations today are struggling with:
+
+- **Siloed information and workflows**: AI can only act on what it can see—often, just a fraction of what's truly relevant.
+- **Complex, expensive integrations**: Every new data source, tool, or AI agent means more custom connectors, patches, and technical debt.
+- **Stunted automation and insight**: When assistants lack rich, real-time access to infrastructure, their utility is massively limited.
+
+## The Solution: AKS-MCP's Open, Secure Protocol
+
+AKS-MCP answers this challenge as a universal, protocol-first bridge between AI agents and AKS. Built on the open Model Context Protocol (MCP), it delivers:
+
+- **One protocol, limitless context**: Replace dozens of brittle integrations with a secure, standard interface—making all your Kubernetes data, events, and actions available to any AI assistant.
+- **Plug-and-play AI agent support**: Works out of the box with Claude, Cursor, GitHub Copilot, and more—so you can use the assistants you already love.
+- **Secure by default**: Leverage Azure authentication and granular RBAC to keep operations and data safe.
+- **Extensible and community-driven**: Fully open source, so you can adapt, extend, and evolve the platform for new tools and future needs.
+
+## Core Capabilities at a Glance
+
+The AKS MCP server supports a number of tools that surface capabilities to interact with Kubernetes and Azure APIs.
+It provides integrations with several monitoring sources to help users diagnose health issues in their AKS clusters and related resources.
+Currently, AKS MCP server supports the following tools:
+
+- **Kubectl commands using the Kubernetes API**: Read and write commands such as `get`, `describe`, `logs`, `exec`, `apply`, `delete` etc.
+- **Azure CLI AKS commands for interacting with Azure APIs**: `show`, `list`, `get-credentials`, nodepool operations, `check-network`
+- **Other commands for AKS-attached Azure resources**: Basic, read-only commands for Azure Compute, Networking, and Kubernetes Fleet Manager
+- **Azure Monitor API commands for retrieving monitoring data**: Pull activity logs, audit logs, metrics, alerts
+- **Other Diagnostic information**: Diagnose and solve detectors, Azure Advisor, [Inspektor Gadget](https://learn.microsoft.com/en-us/troubleshoot/azure/azure-kubernetes/logs/capture-system-insights-from-aks?tabs=azurelinux30) for real time troubleshooting 
```

Comment:
- I think a few user journeys here as samples of what's possible will help land the value prop better.
- nit: need to remove en-us